expressEngine/product/service/product.js

```diff
@@ -25,6 +26,29 @@ class Service {
     }
     return { data: product, message: success.productRegistered };
   }
+
+  updateProduct = async (data) => {
+    const { criteria, dataToUpdate } = data;
+    if (!criteria) {
+      return { error: error.emptyCriteria };
+    }
+    if (!dataToUpdate) {
```

Comment:
@aman-successive Showing success message even when dataToUpdate is empty or have the wrong field.

Reply:
updated it